ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupDemoController.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import FluentUI
+import UIKit
+
+// MARK: - AvatarGroupDemoController
+
+class AvatarGroupDemoController: DemoController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        addTitle(text: "Avatar Stack")
+        insertAvatarViews(style: .stack, showBorders: false)
+        addTitle(text: "Avatar Pile")
+        insertAvatarViews(style: .pile, showBorders: false)
+    }
+
+    private func insertAvatarViews(style: MSFAvatarGroupStyle, showBorders: Bool) {
+        var constraints: [NSLayoutConstraint] = []
+
+        for size in MSFAvatarSize.allCases.reversed() {
+            let containerView = UIView(frame: .zero)
+
+            let avatarGroup = MSFAvatarGroup(style: style, size: size)
+            for index in 0...2 {
```

Comment:
"for index in 0...2" seems a bit odd to me. Is there a reason we don't want to do all of the sample personas, or pass in the number of sample personas we'd like to use?

Reply:
I'm eventually going to add in a maxAvatarCount + overflowCount to determine how many personas to iterate over (not limited to just 3). I should have specified that, sorry!